deno/gateway/v10.ts

```diff
@@ -709,7 +712,9 @@ export type GatewayEntitlementModifyDispatch = DataPayload<
 /**
  * https://discord.com/developers/docs/topics/gateway-events#entitlement-create
  */
-export type GatewayEntitlementCreateDispatchData = GatewayEntitlementModifyDispatchData;
+export type GatewayEntitlementCreateDispatchData = Omit<GatewayEntitlementModifyDispatchData, 'ends_at'> & {
+	ends_at: GatewayEntitlementModifyDispatchData['ends_at'] | null;
```

Comment:
mmm, doesn't rest also return this as possibly nullable? We should add it to the main APIEntitlement type if so